guides/nestor/ISPsetup.md

```diff
@@ -0,0 +1,317 @@
+
+- Set the `Hostname` of you VM machine to whatever you want
+
+![Section9](https://i.imgur.com/isHnTuW.png)
+
+- Set the full name for `new user`
+
+![Section9](https://i.imgur.com/SeJZZFJ.png)
+
+![Section9](https://i.imgur.com/mmDpJK9.png)
+
+- Then set your `Username` for your account
+
+![Section9](https://i.imgur.com/zuIqs7E.png)
+
+- Set the new user `password`
+
+![Section9](https://i.imgur.com/t4L1H5v.png)
+
+- verify password again
+
+![Section9](https://i.imgur.com/Ypjs1pV.png)
+
+- Select if you want to encrypt your home directory `I advice to select no for this test lab`
+
+![Section9](https://i.imgur.com/Tfju0kx.png)
+
+- Select your `Time zone` if the one shown is correct press yes otherwise press no
+
+![Selection9](https://i.imgur.com/FHvvLvn.png)
+
+- Select Partition disks select the `default` the one on the picture.
+
+![Selection9](https://i.imgur.com/ed9XOYY.png)
+
+- Select disk partition the `default`
+
+![Selection9](https://i.imgur.com/VGm3bD6.png)
+
+- On the next step select `YES` on configure LVM
+
+![Selection9](https://i.imgur.com/btRTEMa.png)
+![Selection9](https://i.imgur.com/K0Xme6H.png)
+
+- Select the partition size `you need to leave it as default`
+
+![Selection9](https://i.imgur.com/TiERK5h.png)
+
+- Press `yes` on write the changes to disks
+
+![Selection9](https://i.imgur.com/eTfOxHK.png)
+
+- `Do not` set a proxy leave black and press continue
+
+![Selection9](https://i.imgur.com/ywLngCl.png)
+
+- The update question is all up to you. Can be setup to `no auto updates` or `install security updates automatically`
+
+![Selection9](https://i.imgur.com/2MzOrqS.png)
+
+- On the `sofware selection` for this lab `leave default`
+
+![Selection9](https://i.imgur.com/325nEqo.png)
+
+- on the install the grub boot loader question select `YES`
+
+![Selection9](https://i.imgur.com/ilxTd3J.png)
+
+- On the last question select `continue`
+
+![Selection9](https://i.imgur.com/az5SGZO.png)
+
+## Ubuntu Server Configuration <a id="Third"></a>
+
+3. After Reboot log in into computer with the `username` you setup early and enter `password` to log in
+
+![Selection9](https://i.imgur.com/HjsqfkV.png)
+![Selection9](https://i.imgur.com/avlp5hu.png)
+
+- First step run `sudo apt-get update` `sudo apt-get upgrade` to make sure the system is up to date. 
+- Enable ipv4 routing on Ubuntu. By default Ubuntu doesn't allow for routing on IP traffic.
+- To get this done you will need to edit `sysctl.conf` which is on the directory /etc/. 
+- This can be done by using nano like so, `sudo nano /etc/sysctl.conf`. 
+- It will require your `root password` since it effect the entire system when you edit this file.
+
+![Selection9](https://i.imgur.com/cWJB3p5.png)
+![Selection9](https://i.imgur.com/nkxml1h.png)
+
+- Go to the line that say `Uncomment the next line to enable packet forwarding for IPv4` and the remove the symbol `#` by removing the `#` sign this will make the program read and use that line to `enable packet forwarding`. 
```

Comment:
Change:
Go to the line that say `Uncomment the next line to enable packet forwarding for IPv4` and the remove the symbol `#` by removing the `#` sign this will make the program read and use that line to `enable packet forwarding`.
to:
Go to the line that say `# Uncomment the next line to enable packet forwarding for IPv4` and the remove the symbol `#` from `#net.ipv4.ip_forward=1`. This will make the program read and use that line to enable packet forwarding.